### DISCUSSION TITLE
feature request: better installation process: ask everything at once, ask about pressing enter at the end, sound for indication of completion

### DISCUSSION THREAD

**Discussion by odarcan** [linuxmint/discussions#590] [Ideas]
1: Instead of making it obligatory to stand by the computer,
make it so one can answer every necessary question, and then be able to step away and come back to a freshly installed mint, or at least at the [press enter to restart] screen

2: In fact, ASK if the user wants to restart at end of install or not (knowing if they started the usb install via bios by permanent change or not to drive priorities)

3: a little two tone beep would be nice to indicate install finished,  low then high note for success, and high then low note for installation error. Doesnt matter if it may be disabled, just attempt.

**Comment by Jopp-gh** [comment]
Fully agree. The installer is "boring": that's a good thing but it could offer some more options to configure a kind of "advanced" setup. Eg.
- configuration of redlight or similar applet for the health-conscious citizen
- configuration of panel layout (top, bottom, both)
- installation of drivers for openCL drivers for digital photography (example: amd cards) and for 3d rendering 
- suggestion of apps for different audiences (office , 2d & 3d,  audio-video, coding, entertainment, web, etc)
- installation of specific drivers useful for gaming (controller support for PS3, PS4 or Xbox -pads)
- installation of "better" drivers for graphic tablets with the ability to setup multiple profiles
- checkmark for GUI effects on/off
- checkmark for keyboard key switch Ctrl <> Win for those who don't use windows keyboards!
- setup of xmodmap to replace certain characters on the keyboard (people around the world speak many languages and also need to adapt to English as well as to multiple other languages sometimes)
- workspace setup (applets, options)
- accessibility setup (eg. screen reader voices download and shortcut), which is ideal for the old, usually people think oldies don't know much of computers
- setup windows behavior (preferences)
- setup of notifications
- checkmark to deactivate certain services started by default by systemd
- checkmark to not install certain "standard apps"
- setup to choose system fonts  
- installation of sys admin apps (Mobo temps, ram usage, CPU usage – one applet/desklet instead of conky ) for those who need this stuff

An installer shouldn't give any errors at all, but ignore them and proceed with a successful installation but log everything. 

Logging tools then, are too obscure for the common user and should be simplified. Correctly only developers and sys admins check logs. How about making logging tools more human readable for common users, so people get the habit to check by themselves and learn at least to seek the right info to be able to post useful questions on the forums?

Inxi is OK, but not enough for error resolution.
Ops pretty long list 😊
